### PR TITLE
[Player-832] Updated IMA library to version 3.7.1

### PR DESCRIPTION
**Pull request by akshaydch11**

### CODE REVIEW

File: ExoPlayerSampleApp/app/build.gradle [hunk 1]
```diff
@@ -89,12 +89,12 @@ dependencies {
...
   compile files('libs/OoyalaFreewheelSDK.jar')
   compile files('libs/OoyalaIMASDK.jar')
-  compile files('libs/ima-android-v330.jar')
+  compile files('libs/ima-android-v371.jar')
   compile files('libs/OoyalaSDK.jar')
   compile 'com.google.android.exoplayer:exoplayer:r1.5.7'
 
 // For development purposes only
-//  compile project(':ooyalaCoreSDK')
+//  compile project(':core')
```
Comment (aeng7) [inline ExoPlayerSampleApp/app/build.gradle:97]: What does this change do?

File: settings.gradle [hunk 2]
```diff
@@ -46,6 +46,6 @@ project(':OoyalaSkinSampleApp').projectDir = new File(settingsDir, './OoyalaSkin
 
 // Developer
 //include ':ooyalaSkinSDK'
-//project(':ooyalaSkinSDK').projectDir=new File(settingsDir, '../ios-skin/sdk/android/ooyalaSkinSDK/')
-//include ':ooyalaCoreSDK'
-//project(':ooyalaCoreSDK').projectDir=new File(settingsDir, '../android-sdk/sdk/ooyalaCoreSDK/')
\ No newline at end of file
+//project(':ooyalaSkinSDK').projectDir=new File(settingsDir, '../native-skin/sdk/android/ooyalaSkinSDK/')
+//include ':core'
+//project(':core').projectDir=new File(settingsDir, '../android-sdk/sdk/core/')
```
Comment (aeng7) [inline settings.gradle:51]: What do these changes do?

Comment (akshaydch11) [reply settings.gradle:51]: While importing sdk repository name is 'core',  'ooyalaCoreSDK' is not relevant. Thats the reason every time you enable this you have to change name 'ooyalaCoreSDK' to 'core' also along with uncommenting. Just made little easier.

File: ExoPlayerSampleApp/app/build.gradle (hunk 1 above)
Comment (akshaydch11) [reply ExoPlayerSampleApp/app/build.gradle:97]: While importing sdk repository name is 'core', 'ooyalaCoreSDK' is not relevant. Thats the reason every time you enable this you have to change name 'ooyalaCoreSDK' to 'core' also along with uncommenting. Just made little easier.

**Comment by brol1dev** [comment]
LGTM!

**Review by brol1dev — APPROVED** [review]

**Pull request merged**